contents/uix/uix.md

```diff
@@ -45,3 +47,65 @@ Just like Software Development, UX Design is an iterative process. It involves m
+  * Gather the tools needed for the project such as wireframing or prototyping [tools](https://medium.com/@Mockplus/the-10-best-wireframing-and-prototyping-tools-for-designers-a808e81ecadf) for creating low to high fidelity products for testing
+  * Depending on whether the project is hypothetical (taking a known product and re-designing it) or client-based (working for businesses or non-profits), discuss and create a project plan with stakeholders
+
+#### Deliverables:
+  * [Personas](https://static1.squarespace.com/static/519deddfe4b07b846eef9842/55915575e4b0a919cf5a0da5/55a5b693e4b0576c4f936a78/1436923548676/11_all_with_eps_brands.jpg?format=750w) - User profiles that are representative of the target audiences
+  * [User Stories](https://i.pinimg.com/originals/00/ac/37/00ac379ca5b9c28de3d56bdea4580e2e.jpg) - Representation of the most important user actions or motivations for each persona
+  * [User Scenarios](http://www.davedoyle.com/prof/portfolio/images/large/petcare_scenario.gif) - Case studies of how users use the products, supplementing User Stories to allow you to better empathize with the users
+  * [User Flows](https://cdn-images-1.medium.com/max/1600/0*aO_6rai_fTnSe5rh.) - Depiction of how users navigate through the product for each scenario
+  
+### Prototyping
+Prototyping stage is where you:
+  * [Test](https://usabilitygeek.com/usability-testing-prototypes/) whether the product flow you came up with is smooth
+  * Create [prototypes](https://www.mockplus.com/blog/post/wireframe-mockup-prototype-selection-of-prototyping-tools), or Minimum Viable Product (MVP), for further testing and refinement
+  * Get a clear sense of how users would interact with the final product
+  
+#### Deliverables:
+  * [Wireframes](https://wireframesketcher.com/samples/YouTube.png) - Depicts a skeleton version of an application, meant to develop a unified vision of content structure instead of focusing on asthetics; may or may not be clickable
+  * [UI elements](https://thumbs.dreamstime.com/z/flat-ui-design-elements-set-icons-buttons-progress-bars-vector-illustration-light-colors-33417705.jpg) - Basic visual components, which can start out basic and increase in quantity/quality with the fidelity of MVP
+  * [Mockups](https://i0.wp.com/brandhorse.com/wp-content/uploads/2016/01/EZ-Frabic-Mobile-App-UX-UI-Design.jpg?resize=960%2C750) - Built upon an agreed version of wireframe with higher fidelity and asthetics
+  
+### User Testing
+User Testing stage is where you:
+  * Should have a clear sense of the user goals at this stage
+  * Use a low or high fidelity prototype to [test with users](http://www.uxbooth.com/articles/usability-testing-dont-guess-test/), to determine the effectiveness of product
```

Comment:
I feel that the link should be more appropriately named. I click the link not knowing what to expect. Since its a link to usability testing maybe you could phrase it in this way:


Use a low or high fidelity prototype during [usability testing]()